query/build.sh

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/env bash
-peg -inline -switch language.peg
+
+peg -inline -switch query/language.peg
```

Comment:
http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in

`DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )` works. A handy hack.